examples/scripts/ct_abel.py

```diff
@@ -0,0 +1,114 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# This file is part of the SCICO package. Details of the copyright
+# and user license can be found in the 'LICENSE.txt' file distributed
+# with the package.
+
+r"""
+Abel Transform Demo
+===================
+
+This is a placeholder for more detailed docs."
+"""
+
+import numpy as np
+
+import jax
+
+import abel
+
+import scico.numpy as snp
+from scico import functional, linop, loss, plot
+from scico.linop.abel import AbelProjector
+from scico.optimize.admm import ADMM, LinearSubproblemSolver
+from scico.util import device_info
+
+
+def dist_map_2D(img_shape, center=None):
+
+    if center == None:
+        center = [img_dim // 2 for img_dim in img_shape]
+
+    coords = [np.arange(0, img_dim) for img_dim in img_shape]
+    coord_mesh = np.meshgrid(*coords, sparse=True, indexing="ij")
+
+    dist_map = sum([(coord_mesh[i] - center[i]) ** 2 for i in range(len(coord_mesh))])
+    dist_map = np.sqrt(dist_map)
+
+    return dist_map
+
+
+def create_french_test_phantom(img_shape, radius_list, val_list, center=None):
+
+    dist_map = dist_map_2D(img_shape, center)
+
+    img = np.zeros(img_shape)
+    for r, val in zip(radius_list, val_list):
+        img[dist_map < r] = val
+
+    return img
+
+
+x_gt = create_french_test_phantom((256, 254), [100, 50, 25], [1, 0, 0.5])
+
+y = abel.Transform(np.array(x_gt), direction="forward", method="daun").transform
+x_inv = abel.Transform(np.array(y), direction="inverse", method="daun").transform
+
+
+x_gt = jax.device_put(x_gt)
+
+A = AbelProjector(x_gt.shape)
+y = A @ x_gt
+y = y + 1 * np.random.normal(size=y.shape).astype(np.float32)
+ATy = A.T @ y
+
+
+"""
+Set up ADMM solver object.
+"""
+λ = 1.71e01  # L1 norm regularization parameter
+ρ = 4.83e01  # ADMM penalty parameter
+maxiter = 100  # number of ADMM iterations
+cg_tol = 1e-4  # CG relative tolerance
+cg_maxiter = 25  # maximum CG iterations per ADMM iteration
+
+g = λ * functional.L1Norm()  # regularization functionals gi
+C = linop.FiniteDifference(input_shape=x_gt.shape)  # analysis operators Ci
+
```

Comment:
I think the comments after the two lines above can be dropped, but if not, functionals and operators should be singular.

Reply:
Removed